game-core/src/main/java/games/strategy/engine/chat/CompositeIcon.java

```diff
@@ -6,6 +6,9 @@
 
 import javax.swing.Icon;
 
+/**
+ * An icon that is rendered as a composite of zero or more other icons.
```

Comment:
> rendered as a composite of zero or more other icons.

that helps, but what does that mean?  Are the icons rendered on top of each other, next to each other?